doc/dynamic-environments/quickstart.mkd

```diff
@@ -3,22 +3,21 @@
 This intent of this document is to serve as a basic guide for getting started with r10k and a fresh Puppet installation, including the following:
 
 * Installing Puppet and its dependencies
```

Comment:
This document implicitly assumes that we're installing Puppet 4, could you call that out here?